multicluster/config/overlays/member/role.yaml

```diff
@@ -0,0 +1,162 @@
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: controller-role
```

Comment:
We could not use a name like "controller-role" which can easily conflicts with other controllers in the cluster. Add "antrea-mc-" prefix for all such K8s standard resources.

Reply:
yes, this is a base file, we add a prefix `antrea-mc-` in kustomization.yml which will add this prefix in final manifest.

Reply:
Ah, I recall this now!